frontend/src/concepts/hardwareProfiles/validationUtils.ts

```diff
@@ -185,204 +181,7 @@ export const isHardwareProfileConfigValid = (data: HardwareProfileConfig): boole
-        if (maxCount !== undefined && !Number.isInteger(maxCount)) {
-          ctx.addIssue({
-            code: z.ZodIssueCode.custom,
-            message: createIdentifierWarningMessage(
-              `Maximum count for ${
-                identifier.resourceType ?? identifier.displayName
-              } cannot be a decimal.`,
-              data.isDefault,
-            ),
-            params: {
-              type: HardwareProfileWarningType.OTHER,
-            },
-          });
-        }
-        if (!Number.isInteger(defaultCount)) {
-          ctx.addIssue({
-            code: z.ZodIssueCode.custom,
-            message: createIdentifierWarningMessage(
-              `Default count for ${
-                identifier.resourceType ?? identifier.displayName
-              } cannot be a decimal.`,
-              data.isDefault,
-            ),
-            params: {
-              type: HardwareProfileWarningType.OTHER,
-            },
-          });
-        }
-        if (maxCount !== undefined && minCount !== undefined && minCount > maxCount) {
-          ctx.addIssue({
-            code: z.ZodIssueCode.custom,
-            message: createIdentifierWarningMessage(
-              `Minimum allowed ${
-                identifier.resourceType ?? identifier.displayName
-              } cannot exceed maximum allowed ${
-                identifier.resourceType ?? identifier.displayName
-              }.`,
-              data.isDefault,
-            ),
-            params: {
-              type: HardwareProfileWarningType.OTHER,
-            },
-          });
-        }
-        if (
-          (defaultCount !== undefined && minCount !== undefined && defaultCount < minCount) ||
-          (maxCount !== undefined && defaultCount !== undefined && defaultCount > maxCount)
-        ) {
-          ctx.addIssue({
-            code: z.ZodIssueCode.custom,
-            message: createIdentifierWarningMessage(
-              `The default count for ${
-                identifier.resourceType ?? identifier.displayName
-              } must be between the minimum allowed ${
-                identifier.resourceType ?? identifier.displayName
-              } and maximum allowed ${identifier.resourceType ?? identifier.displayName}.`,
-              data.isDefault,
-            ),
-            params: {
-              type: HardwareProfileWarningType.OTHER,
-            },
-          });
-        }
-      } catch (e) {
-        ctx.addIssue({
-          code: z.ZodIssueCode.custom,
-          message: createIdentifierWarningMessage(
-            `The resource count for ${
-              identifier.resourceType ?? identifier.displayName
-            } has an invalid unit.`,
-            data.isDefault,
-          ),
-          params: {
-            type: HardwareProfileWarningType.OTHER,
-          },
-        });
-      }
-    }
-  });
+export const hardwareProfileWarningSchema = z.object({
+  isDefault: z.boolean(),
+  value: z.array(identifierSchema),
+});
```

Comment:
why still have this one at all anymore. can we just use the other one?

Reply:
If we're removing isDefault then it's fine removing this, and continue with the other one.